### PR TITLE
Protobuf change and action functions for scenario

### PR DESCRIPTION
**Pull request by cj-chung**
1. Protobuf definition change - add tenant_id and project_id for InternalVpcInfo
2. add entities for scenario actions and event
3. add status for each service
4. ignore some entity's fields from swagger
5. add deploy functions in scenario

### CODE REVIEW

**Review by phudtran — APPROVED** [review]
LGTM

**Comment by yanmo96** [comment]
LGTM

**Review by yanmo96 — APPROVED** [review]
LGTM

File: services/scenario-manager/entities/entities.go [hunk 1]
```diff
@@ -66,20 +73,21 @@ type Service struct {
...
-	VhostsPerRack    uint      `json:"vhosts_per_rack"`
-	DataPlaneCidr    string    `json:"data_plane_cidr"`
-	NumberOfGateways uint      `json:"number_of_control_plane_gateways"`
-	GatewayIPs       []string  `json:"control_plane_gateway_ips"`
-	Images           []Image   `json:"images"`
-	VNodes           []VNode   `json:"vnodes"`
-	VLinks           []VLink   `json:"vlinks"`
-	CreatedAt        time.Time `json:"created_at"`
-	UpdatedAt        time.Time `json:"updated_at"`
+	Id               string        `json:"id" swaggerignore:"true"`
```
Comment (zzengcs) [inline services/scenario-manager/entities/entities.go:76]: What 's the difference for swaggerignore between here and status in the later parts?

**Review by zzengcs — APPROVED** [review]
LGTM

**Pull request merged**